### PR TITLE
[ADD][10.0] medical_app_pharmacy

### PR DESCRIPTION
**Pull request by lasley**
* Create app to provide contextual menus for pharmacies

Depends:
- [x] #34

### CODE REVIEW

**Comment by lasley** [comment]
Rebased; ready for review

**Review by tedsalmon — CHANGES_REQUESTED** [review]
Tests are failing

```
"External ID not found in the system: medical_app_pharmacy.medical_patient_action" while parsing /home/travis/build/laslabs/vertical-medical/medical_app_pharmacy/views/pharmacy_menu.xml:22, near
<menuitem id="medical_patient" name=" Patient" parent="medical_patient_root" sequence="10" action="medical_patient_action"/>
```
Additionally, there is this warning:
```
2016-11-22 04:51:25,804 5354 WARNING openerp_test odoo.modules.module: module pharmacy: module not found
```

**Comment by lasley** [comment]
Oops I don't think I meant to remove WIP tag from this. This is definitely not done - SMD needs to provide input

File: medical_app_pharmacy/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1 @@
...
        'medical_medicament_storage',
        'sale_crm_medical_prescription',
        'sale_stock_medical_prescription',
    ],
    'data': [
        'templates/assets.xml',
        'views/pharmacy_menu.xml',
        'data/pharmacy_tour.xml',
    ],
}
```
Comment (woodbrettm) [inline medical_app_pharmacy/__manifest__.py]: No newline and some odd character issues.

**Comment by obulkin** [comment]
I've addressed the comment by @bmw95 and made a number of other small improvements and fixes. This PR should now be ready for another round of review.

**Comment by lasley** [comment]
LGTM thanks Oleg

**Pull request merged**